src/index.ts

```diff
@@ -110,6 +129,7 @@ export default function supernova(env: Galaxy) {
               rect,
               footerContainer,
               announce,
+              model,
```

Comment:
why are you adding model here?

Reply:
That is likely some left over from the prototypes I made. Will remove it.